src/main/java/duke/exception/ErrorMessage.java

```diff
@@ -0,0 +1,56 @@
+package duke.exception;
+
+public class ErrorMessage {
+
+    public static void printDeadlineSyntaxCommandMessage(String command) {
+        System.out.println("-----------------------------------------");
```

Comment:
Can try declaring line separator as constant so you don’t have to keep copy and pasting the same print ——- line